package.json

```diff
@@ -69,7 +69,7 @@
     "rollup-plugin-delete": "^2.0.0",
     "rollup-plugin-glob-import": "^0.4.5",
     "rollup-plugin-peer-deps-external": "^2.2.3",
-    "rollup-plugin-postcss": "^3.1.8",
```

Comment:
tbh I cannot find any reference to that version in their repo. Tried to build all our packages and it didn't cause any errors - should be good imo

Reply:
not sure what you mean, why not publish nebenan-rollup-plugin-postcss@1.0.0

Reply:
https://github.com/egoist/rollup-plugin-postcss/blob/master/package.json

no branch in their repo mentions version 3.1.8

I used 3.0.1 to keep their version scheme - would be weird to update later from nebenan-rollup-plugin-postcss 1.0.0 to rollup-plugin-postcss 3.1.x

Reply:
why? it's a different package. it's all good

Reply:
is it worth changing it? would need to unrelease + re-release

Reply:
i think it's better to avoid confision